### PR TITLE
complete task 6

### PR DESCRIPTION
**Pull request by DmitryGhoncharko**

### CODE REVIEW

**Review by AntonNovosad — APPROVED** [review]

File: src/task6/School.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    public void printAllStudiesWhereAverageScopeMoreThanMiddle() {
+        if (averageScore == 0) {
+            averageScore = calculateAverageScopeAllStudents();
+        }
+        for (Student student : students) {
+            if (student.getAverageScore() < averageScore) {
+                System.out.println(student);
+            }
+        }
+
```
Comment (AlexeyKatomachin) [inline src/task6/School.java:40]: remove empty row

File: src/task6/School.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
```
Comment (AlexeyKatomachin) [inline src/task6/School.java:42]: remove empty row

File: src/task6/School.java (hunk 1 above)
Comment (DmitryGhoncharko) [reply src/task6/School.java:40]: fixed

File: src/task6/School.java (hunk 2 above)
Comment (DmitryGhoncharko) [reply src/task6/School.java:42]: fixed

**Review by AntonNovosad — APPROVED** [review]

**Review by AlexeyKatomachin — APPROVED** [review]

**Pull request merged**